wagtailio/project_styleguide/templates/patterns/components/streamfields/quotes.html

```diff
@@ -0,0 +1,21 @@
+{% load wagtailcore_tags %}
```

Comment:
this is.. out of place?

Reply:
You're right, I didn't see this on the PR, I'm genuinely surprised how it even is here.